code/modules/mob/living/carbon/xenomorph/castes/hivemind/hivemind.dm

```diff
@@ -0,0 +1,126 @@
+/mob/living/carbon/xenomorph/hivemind
+	caste_base_type = /mob/living/carbon/xenomorph/hivemind
+	name = "Hivemind"
+	real_name = "Hivemind"
+	desc = "A glorious singular entity."
+
+	icon_state = "hivemind_marker"
+	icon = 'icons/mob/cameramob.dmi'
+
+	status_flags = GODMODE | UNDETECTABLE
+	density = FALSE
+	throwpass = TRUE
+
+	health = 1000
+	maxHealth = 1000
+	plasma_stored = 5
+	tier = XENO_TIER_ZERO
+	upgrade = XENO_UPGRADE_ZERO
+
+	mouse_opacity = MOUSE_OPACITY_OPAQUE
+	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
+	see_invisible = SEE_INVISIBLE_LIVING
+	invisibility = INVISIBILITY_OBSERVER
+	sight = SEE_MOBS|SEE_TURFS|SEE_OBJS
+	see_in_dark = 8
+	move_on_shuttle = TRUE
+
+	hud_type = /datum/hud/hivemind
+	hud_possible = list(PLASMA_HUD, PHEROMONE_HUD, QUEEN_OVERWATCH_HUD)
+
+	var/mob/living/carbon/xenomorph/observed_xeno
+
+/mob/living/carbon/xenomorph/hivemind/Initialize(mapload, ...)
+	. = ..()
+	GLOB.hivemind_list += src
+
+	// Remove some unused xeno things
+	verbs -= /mob/living/proc/lay_down
```

Comment:
Could you turn the verb addition into a proc to be redefined at the hivemind level, instead of adding and removing it?

Reply:
This one is still a thing.